pom.xml

```diff
@@ -108,11 +103,6 @@
 		<slf4j.version>1.7.2</slf4j.version>
 		<xmlunit.version>1.5</xmlunit.version>
 		<woodstox.version>4.4.0</woodstox.version>
-
-		<!-- Default versions for TDS projects -->
-		<tds-shared.version>1.0.1-BUILD-SNAPSHOT</tds-shared.version>
-		<tds-dll.version>0.0.1-SNAPSHOT</tds-dll.version>
```

Comment:
Removed the tds-dll (TDS_TestDeliverySystemDataAccess) dependency from this project.
Replaced the tds-shared.version property with references to \${project.version}